project/Game.py

```diff
@@ -170,6 +174,24 @@ def modify_landmark(self, oldclue, clue=None, question=None, answer=None):
         except KeyError:
             return False
 
+    def edit_landmark_order(self, oldindex, newindex):
+        if not self.started:
+            try:
+                index1 = int(oldindex)
```

Comment:
I don't like checking for type in game. We should treat this as if the parameter was type defined as int and rely on CLI to pass the right thing (it casts to int as needed). If somehow something else gets passed then we rightly crash (and caller (cli) can handle it if needed)

Reply:
That's not a blocker, just a style thing.